docker-compose.yml

```diff
@@ -1,102 +1,28 @@
-#       - HTTPS_PROXY=http://proxy.domain.com
-    depends_on:
-      - mongo
-    ports:
-      - 3000:3000
-    labels:
-      - "traefik.backend=rocketchat"
-      - "traefik.frontend.rule=Host: your.domain.tld"
-
-  mongo:
-    image: mongo:4.0
-    restart: unless-stopped
-    volumes:
-     - ./data/db:/data/db
-     #- ./data/dump:/dump
-    command: mongod --smallfiles --oplogSize 128 --replSet rs0 --storageEngine=mmapv1
-    labels:
-      - "traefik.enable=false"
+volumes:
+  packages:
 
-  # this container's job is just run the command to initialize the replica set.
-  # it will run the command and remove himself (it will not stay running)
-  mongo-init-replica:
-    image: mongo:4.0
-    command: >
-      bash -c
-        "for i in `seq 1 30`; do
-          mongo mongo/rocketchat --eval \"
-            rs.initiate({
-              _id: 'rs0',
-              members: [ { _id: 0, host: 'localhost:27017' } ]})\" &&
-          s=$$? && break || s=$$?;
-          echo \"Tried $$i times. Waiting 5 secs...\";
-          sleep 5;
-        done; (exit $$s)"
-    depends_on:
-      - mongo
-
-  # hubot, the popular chatbot (add the bot user first and change the password before starting this image)
-  hubot:
-    image: rocketchat/hubot-rocketchat:latest
-    restart: unless-stopped
-    environment:
-      - ROCKETCHAT_URL=rocketchat:3000
-      - ROCKETCHAT_ROOM=GENERAL
-      - ROCKETCHAT_USER=bot
-      - ROCKETCHAT_PASSWORD=botpassword
-      - BOT_NAME=bot
-  # you can add more scripts as you'd like here, they need to be installable by npm
-      - EXTERNAL_SCRIPTS=hubot-help,hubot-seen,hubot-links,hubot-diagnostics
-    depends_on:
-      - rocketchat
-    labels:
-      - "traefik.enable=false"
+services:
+  build:
+    # see compose v2 bug - https://github.com/docker/compose/issues/8785
+    # see fix PR - https://github.com/docker/compose/pull/8793
+    image: rocket.chat.builder
+    build:
+      context: .docker
+      dockerfile: Dockerfile.meteor
+      args:
+        - RELEASE=2.2
     volumes:
-      - ./scripts:/home/hubot/scripts
-  # this is used to expose the hubot port for notifications on the host on port 3001, e.g. for hubot-jenkins-notifier
-    ports:
-      - 3001:8080
-
-  #traefik:
-  #  image: traefik:latest
-  #  restart: unless-stopped
-  #  command: >
-  #    traefik
-  #     --docker
-  #     --acme=true
-  #     --acme.domains='your.domain.tld'
-  #     --acme.email='your@email.tld'
-  #     --acme.entrypoint=https
-  #     --acme.storagefile=acme.json
-  #     --defaultentrypoints=http
-  #     --defaultentrypoints=https
-  #     --entryPoints='Name:http Address::80 Redirect.EntryPoint:https'
-  #     --entryPoints='Name:https Address::443 TLS.Certificates:'
-  #  ports:
-  #    - 80:80
-  #    - 443:443
-  #  volumes:
-  #    - /var/run/docker.sock:/var/run/docker.sock
+      - type: bind
+        source: .
+        target: /mnt
+      - type: volume
+        source: packages
```

Comment:
Should this instead be .meteor/packages as well.  That way it's not another folder to ignore in gitignore?

Reply:
I'm not sure what you mean. I'm not yet a 100% well versed in meteor build system. The only data that I found needed to be saved across runs was `$HOME/.meteor/packages`.

Reply:
I'm not sure what you mean